lib/arc/arc.dart

```diff
@@ -0,0 +1,150 @@
+// ignore_for_file: sort_constructors_first, cascade_invocations
+
+import 'dart:async';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:archive/archive.dart';
+import 'package:flutter_storm/flutter_storm.dart';
+import 'package:flutter_storm/flutter_storm_defines.dart';
+import 'package:retro/utils/log.dart';
+
+enum ArcMode {
+  otr, o2r
+}
+
+class Arc {
```

Comment:
nitpick(naming): i'd prefer `Archive` to `Arc` but that shouldn't block merging

Reply:
Im gonna think about it, but i love Arc tbh xD